applications_tests/lethe-fluid-particles/restart_fluidized_bed_files/dem.triangulation.info

```diff
@@ -1,2 +1,2 @@
 version nproc n_attached_fixed_size_objs n_attached_variable_size_objs n_coarse_cells
-5 1 0 1 25
```

Comment:
Do you have an idea why this number changed? Anything significant happen or change in the mesh? It's not a big deal.II and also could be a p4est version issue.

Reply:
Git does not compare the same file, anyway, as Oreste pointed out, those dem files are not used, I forgot to delete them